MKDOCS_MATERIAL_MERGE_BASE

```diff
@@ -1 +1 @@
-ccbf5a693cae34065d2997a1af37992a148adddf
+4f641cc0ee7befd440956e53b8798899482f6ed4
```

Comment:
This commit hasn't been released as stable. I'm worried if this might merge in unaddressed bugs that haven't been fleshed out before upstream's next release.

I'm not asking you to undergo another nightmare of re-merging in this instance, but it might be better in the future to merge in a stable release's tagged commit.

Reply:
That's a good point --- can upgrade again when there is a new stable release --- that should be fairly easy.